### ISSUE TITLE
Account is shown twice

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#133]
### Actual behaviour
- fresh installation
- adding account
- account is shown twice in account list

### Expected behaviour
- show account only once
 
### Steps to reproduce
1. fresh installation
2. adding account
3. account is shown twice in account list

**Comment by przybylski** [comment]
nextcloud and owncloud simultanously installed?

**Comment by tobiasKaminsky** [comment]
No, it is an emulator with only nextcloud, but also nextcloud-beta.
Do you think this is a problem?

**Comment by AndyScherzinger** [comment]
Haven't had this behavior or at least didn't recognize it. Will have a look.
Is this a 1.2.0 thing or could we postpone it to 1.2.1 or 1.3.0 (or 1.2.0 after an RC1 for example)?

**Comment by tobiasKaminsky** [comment]
currently master I guess.
We can fix it while doing RC1 as it is a minor bug

**Comment by przybylski** [comment]
I cannot reliably reproduce, I did saw this issue once

**Comment by AndyScherzinger** [comment]
should be fixed in 1.3.0 with the change that the active account isn't shown in the list at all :)
